sklearn/decomposition/tests/test_nmf.py

```diff
@@ -589,12 +707,66 @@ def test_nmf_custom_init_dtype_error():
     W = rng.random_sample((20, 15))
 
     with pytest.raises(TypeError, match="should have the same dtype as X"):
-        NMF(init='custom').fit(X, H=H, W=W)
+        Estimator(init='custom').fit(X, H=H, W=W)
 
     with pytest.raises(TypeError, match="should have the same dtype as X"):
         non_negative_factorization(X, H=H, update_H=False)
 
 
+def test_nmf_is_minibatch_nmf():
+    # Test that the standard nmf is the minibatch nmf after 1 iteration
+    # with batch_size = n_samples and forget_factor 0.0
+    rng = np.random.mtrand.RandomState(42)
+    X = np.abs(rng.randn(48, 5))
+    max_iter = 1
+    init = 'nndsvda'  # FIXME : should be removed in 1.1
+    nmf = NMF(5, solver='mu', init=init, random_state=0,
+              max_iter=max_iter,)
+    mbnmf = MiniBatchNMF(5, solver='mu', init=init, random_state=0,
+                         max_iter=max_iter,
+                         batch_size=X.shape[0], forget_factor=0.0)
+    W = nmf.fit_transform(X)
+    mbW = mbnmf.fit_transform(X)
+    assert_array_almost_equal(W, mbW)
```

Comment:
```suggestion
    assert_allclose(W, mbW)
```

Reply:
I have chenged to `asset_allclose` all the test that I have modified. When NMF was tested in the same test, I have changed for consistency too.